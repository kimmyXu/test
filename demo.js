export const generateConfig = () => {
   return {
      server: 'http://aaa.com',
      port: 80812,
      domain: 'localhost',
      time: new Date
   }
}


export const generateAnotherConfig = () => {
   return {
      server: 'http://aaa.com',
      port: 80811,
      domain: 'localhost',
      time: new Date
   }
}