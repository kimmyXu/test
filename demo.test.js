import { generateConfig, generateAnotherConfig } from "./demo";

test('测试 generateConfig 函数', ()=>{
    // expect(generateConfig()).toEqual(
    //     {
    //         server: 'http://aaa.com',
    //         port: 8080,
    //         domain: 'localhost',
    //     }
    // )
    expect(generateConfig()).toMatchSnapshot({
        time: expect.any(Date)
    });
})

test('测试 generateAnotherConfig 函数', ()=>{
    expect(generateAnotherConfig()).toMatchSnapshot({
        time: expect.any(Date)
    });
})

