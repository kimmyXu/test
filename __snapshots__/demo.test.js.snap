// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 generateAnotherConfig 函数 1`] = `
Object {
  "domain": "localhost",
  "port": 80811,
  "server": "http://aaa.com",
  "time": Any<Date>,
}
`;

exports[`测试 generateConfig 函数 1`] = `
Object {
  "domain": "localhost",
  "port": 80812,
  "server": "http://aaa.com",
  "time": Any<Date>,
}
`;
